### PR TITLE
[ Fabric ] Update feature/fabric220 with feature/helm3 branch

### PR DESCRIPTION
**Pull request by lakshyakumar**
**Changelog**
- update feature/fabric220 with feature/helm

 

**Reviewed by**
@suvajit-sarkar @jagpreetsinghsasan

### CODE REVIEW

**Comment by sownak** [comment]
There are conflicts

**Review by lakshyakumar — COMMENTED** [review]
Fabric code looks fine

File: platforms/r3-corda-ent/configuration/roles/helm_component/templates/idman.tpl [hunk 1]
```diff
@@ -1,10 +1,6 @@
-apiVersion: flux.weave.works/v1beta1
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/configuration/roles/helm_component/templates/idman.tpl:1]: This looks like a mistake have to revert this back

File: platforms/r3-corda-ent/configuration/roles/helm_component/templates/float.tpl [hunk 2]
```diff
@@ -1,10 +1,6 @@
-apiVersion: flux.weave.works/v1beta1
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/configuration/roles/helm_component/templates/float.tpl:1]: This is incorrect merge : change this file to what we have in feature/helm3 branch.

File: platforms/r3-corda-ent/configuration/roles/helm_component/templates/idman.tpl (hunk 1 above)
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/configuration/roles/helm_component/templates/idman.tpl:1]: This is incorrect merge : change this file to what we have in feature/helm3 branch.

File: platforms/r3-corda-ent/configuration/roles/helm_component/templates/nmap.tpl [hunk 3]
```diff
@@ -1,18 +1,10 @@
-apiVersion: flux.weave.works/v1beta1
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/configuration/roles/helm_component/templates/nmap.tpl:1]: This is incorrect merge : change this file to what we have in feature/helm3 branch.

File: platforms/r3-corda-ent/configuration/roles/helm_component/templates/node.tpl [hunk 4]
```diff
@@ -1,10 +1,6 @@
-apiVersion: flux.weave.works/v1beta1
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/configuration/roles/helm_component/templates/node.tpl:1]: This is incorrect merge : change this file to what we have in feature/helm3 branch.

File: platforms/r3-corda-ent/configuration/roles/helm_component/templates/notary.tpl [hunk 5]
```diff
@@ -1,10 +1,6 @@
-apiVersion: flux.weave.works/v1beta1
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/configuration/roles/helm_component/templates/notary.tpl:1]: This is incorrect merge : change this file to what we have in feature/helm3 branch.

File: platforms/r3-corda-ent/configuration/roles/helm_component/vars/main.yaml [hunk 6]
```diff
@@ -1,6 +1,7 @@
 helm_templates:
   pki-generator: pki-generator.tpl
-  signer: create_signer.tpl
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/configuration/roles/helm_component/vars/main.yaml:3]: remove the corresponding actual file aswell

**Review by suvajit-sarkar — APPROVED** [review]
LGTM now

**Review by jagpreetsinghsasan — APPROVED** [review]
I am not able to review such big changelog. As this is BAF branch to branch PR, I will consider it safe to merge

**Pull request merged**